.github/workflows/system-tests-centos7.yml

```diff
@@ -21,12 +21,22 @@ jobs:
           - 5000:5000
     steps:
       - uses: actions/checkout@main
+      - name: Get changed files
+        id: changed-files-excluding-tests
+        uses: tj-actions/changed-files@v35
+        with:
+          files_ignore: |
```

Comment:
ignored files are repeated, could you use a variabile to avoid it?

Reply:
As far as I know, there is no way to define a variable in the code and share it across workflows.
However, this is just the first stab, we can improve later.